prow/plank/plank_test.go

```diff
@@ -690,6 +690,11 @@ func TestBatch(t *testing.T) {
 	if fc.prowjobs[0].Status.State != kube.FailureState {
 		t.Fatalf("Wrong state: %v", fc.prowjobs[0].Status.State)
 	}
+
+	// This is what the SQ reads.
+	if fc.prowjobs[0].Spec.Context != "Some Job Context" {
+		t.Fatalf("Wrong context: %v", fc.prowjobs[0].Spec.Context)
```

Comment:
Personally I believe this kind of error message ought to say both the actual and expected, but I realize none of the other tests here do that